runners/samza/src/main/java/org/apache/beam/runners/samza/runtime/SamzaTimerInternalsFactory.java

```diff
@@ -279,18 +302,15 @@ public void setTimer(TimerData timerData) {
       // persist it first
       state.persist(keyedTimerData);
 
-      // TO-DO: apply the same memory optimization over processing timers
+      // add timer data to buffer where applicable
       switch (timerData.getDomain()) {
         case EVENT_TIME:
           /*
            * To determine if the upcoming KeyedTimerData could be added to the Buffer while
            * guaranteeing the Buffer's timestamps are all <= than those in State Store to preserve
            * timestamp eviction priority:
            *
-           * <p>1) If maxEventTimeInBuffer == long.MAX_VALUE, it indicates that the State is empty,
-           * therefore all the Event times greater or lesser than newTimestamp are in the buffer;
-           *
-           * <p>2) If newTimestamp < maxEventTimeInBuffer, it indicates that there are entries
```

Comment:
Why is this deleted? Does it no longer apply for event time is it?

Reply:
This no longer represent the below code completely changed (even before my change, removed 1) and kept 2)